gnucash/gnome/dialog-price-editor.c

```diff
@@ -192,6 +194,56 @@ price_to_gui (PriceEditDialog *pedit_dialog)
 }
 
 
+static gboolean
+pedit_dialog_replace_found_price (PriceEditDialog *pedit_dialog,
+                                  const gnc_commodity *commodity,
+                                  const gnc_commodity *currency, time64 t)
+{
+    gboolean price_found = FALSE;
+    GNCPrice *test_price =  gnc_pricedb_lookup_day_t64(pedit_dialog->price_db,
+                                commodity, currency, t);
```

Comment:
Whitespace: Add a space between the function and the (, align commodity with pedit_dialog.

Reply:
I must of missed this, normally I would add the space on parts I touch and leave the rest in the file alone, maybe it was like that when I copied and pasted, changed.